### PR TITLE
Documenting 2050824 in main body of text where deemed relevant

### PR DESCRIPTION
**Pull request by kquinn1204**
[TELCODOCS-1016](https://issues.redhat.com//browse/TELCODOCS-1016): Deleting a BFDProfile object does not delete the corresponding running-conf from the MetalLB speakers

See also https://bugzilla.redhat.com/show_bug.cgi?id=2050824

Version(s):
4.10

Preview: http://file.emea.redhat.com/kquinn/TELCODOCS-1016/networking/metallb/metallb-configure-bgp-peers.html#nw-metallb-example-specify-bfd-profile_configure-metallb-bgp-peers

### CODE REVIEW

**Comment by kquinn1204** [comment]
@cgoncalves @sabinaaledort can you give an LGTM to this PR.

File: modules/nw-metallb-example-bgppeer.adoc [hunk 1]
```diff
@@ -48,6 +48,11 @@ spec:
   holdTime: "10s"
   bfdProfile: doc-example-bfd-profile-full
 ----
+//Dependency on RHEL bug 2054160 being addressed.Remove note when fixed.
+[NOTE]
+====
+Deleting the bidirectional forwarding detection (BFD) profile and removing the `bfdProfile` added to the border gateway protocol (BGP) peer resource does not disable the BFD. Instead, the BGP peer starts using the default BFD profile. To disable BFD from a BGP peer resource, delete the BGP peer configuration and recreate it without a BFD profile.
```
Comment (cgoncalves) [inline modules/nw-metallb-example-bgppeer.adoc:54]: Could we reference https://bugzilla.redhat.com/show_bug.cgi?id=2050824 ?

File: release_notes/ocp-4-10-release-notes.adoc [hunk 2]
```diff
@@ -2518,7 +2518,7 @@ In the table below, features are marked with the following statuses:
 // TODO: This known issue should carry forward to 4.8 and beyond! This needs some SME/QE review before being updated for 4.9.
 * In {product-title} 4.1, anonymous users could access discovery endpoints. Later releases revoked this access to reduce the possible attack surface for security exploits because some discovery endpoints are forwarded to aggregated API servers. However, unauthenticated access is preserved in upgraded clusters so that existing use cases are not broken.
 +
-If you are a cluster administrator for a cluster that has been upgraded from {product-title} 4.1 to 4.8, you can either revoke or continue to allow unauthenticated access. It is recommended to revoke unauthenticated access unless there is a specific need for it. If you do continue to allow unauthenticated access, be aware of the increased risks.
```
Comment (cgoncalves) [inline release_notes/ocp-4-10-release-notes.adoc:2521]: This is an unrelated change to the MetalLB BFD note. If needed, this should be done in a separate PR.

**Review by cgoncalves — CHANGES_REQUESTED** [review]
This applies to OCP 4.12 down to 4.10. Please submit PR against the main branch and then run with the backports.

File: modules/nw-metallb-example-bgppeer.adoc (hunk 1 above)
Comment (kquinn1204) [reply modules/nw-metallb-example-bgppeer.adoc:54]: OK was just thinking the same

File: release_notes/ocp-4-10-release-notes.adoc (hunk 2 above)
Comment (kquinn1204) [reply release_notes/ocp-4-10-release-notes.adoc:2521]: OK will remove this I just noticed this kept referring to 4.8 even when release version was rev'd

**Review by cgoncalves — CHANGES_REQUESTED** [review]
The changes look better now. However, as I suggested previously, you need to first propose this change against the main branch and then backport -- implies obsoleting this PR.

**Pull request closed without merging**